app/Mage.php

```diff
@@ -924,9 +924,11 @@ public static function log($message, $level = null, $file = '', $forceLog = fals
             }
 
             if (is_array($message) || is_object($message)) {
+                // phpcs:ignore Ecg.Security.ForbiddenFunction.Found
                 $message = print_r($message, true);
```

Comment:
What is the point of forbidden functions when it can be ignored?

Reply:
Please ignore it.  Most of them are reverted and removed in https://github.com/OpenMage/magento-lts/pull/4303.

Reply:
Wait ... better fix it now.